### PR TITLE
Fixed minor bugs when not using demo data.

### PR DESCRIPTION
**Pull request by CHelfgott**

### CODE REVIEW

File: routes/api/queries/dhis/Network.js [hunk 1]
```diff
@@ -304,11 +304,15 @@ function fetchData(organizationDetails) {
     organizationId,
     rootNode,
   } = organizationDetails;
-  return makeRequest(auth, baseURL, indicatorDict)
-    .then(data => interpretStats(data, indicatorDict))
+  return makeRequest(auth || {}, baseURL, indicatorDict)
+    .catch(() => {})
```
Comment (klandell) [inline routes/api/queries/dhis/Network.js]: Why don't you make this line return an object with an empty rows array instead.
```suggestion
    .catch(() => ({ rows: []}))
```

File: routes/api/queries/dhis/Network.js [hunk 2]
```diff
@@ -304,11 +304,15 @@ function fetchData(organizationDetails) {
...
+    .then(data => {
```
Comment (klandell) [inline routes/api/queries/dhis/Network.js]: You can then get rid of the changes to this block.

File: routes/api/queries/dhis/Network.js (hunk 1 above)
Comment (CHelfgott) [reply routes/api/queries/dhis/Network.js]: Done

File: routes/api/queries/dhis/Network.js (hunk 2 above)
Comment (CHelfgott) [reply routes/api/queries/dhis/Network.js]: Done

**Pull request merged**